lib/app.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const express = require('express');
+const info = require('./server_postgreSQL_info');
+const database = require('./database');
+const stats = require('./stats');
+
+module.exports = function start(callback) {
+  var dbData = database();
+  var Player = dbData.definePlayer();
+  var my = info();
+  var app = express();
+  app.use('/player', stats() );
+  var server = app.listen(process.env.PORT || my.expressServer, function() {
+    console.log('server is connected');
+    Player.sync({force: true}).then(function(){
```

Comment:
in a real app, you wouldn't want to use `force: true` as it will delete existing data!